static/styles/night_mode.scss

```diff
@@ -151,7 +151,7 @@ body.night-mode .compose_table .recipient_box:focus {
 body.night-mode .message-header-contents,
 body.night-mode .message_header_private_message .message-header-contents,
 body.night-mode #tab_list li.active {
-    background: hsla(0, 0%, 0%, 0.2);
+    background: #1a242f;
```

Comment:
As a policy, we specify all colors in Zulip use `hsla`, as part of making the "night mode" feature reasonable to support.  Check out `night_mode.scss` to get a feel for how this works (and visit "display settings" for how to toggle it).  

Most use of "transparent" colors in Zulip is to avoid needing to have duplicate CSS for huge numbers of components.